.changeset/itchy-balloons-call.md

```diff
@@ -0,0 +1,5 @@
+---
+"@turnkey/telegram-cloud-storage-stamper": minor
+---
+
+Initial release
```

Comment:
let's add a bit more context, or you can point it to the README if you'd like to consolidate the commentary there

Reply:
added!